temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivityWorker.java

```diff
@@ -233,10 +288,17 @@ private ActivityTaskHandler.Result handleLocalActivity(
 
       // For small backoff we do local retry. Otherwise we will schedule timer on server side.
       // TODO(maxim): Use timer queue for retries to avoid tying up a thread.
```

Comment:
Did you find where this implementation actually is, if anywhere?

Reply:
It was actually in the code base but was removed during fat finger refactoring more than 2 years ago. https://github.com/temporalio/sdk-java/issues/1261
Will need to bring it back.